src/Queen.cpp

```diff
@@ -0,0 +1,17 @@
+#include "Queen.h"
+
+Queen::Queen(bool isWhite)
+	:Piece(isWhite)
+{
+}
+
+bool Queen::isCorrectMove(const Location& source, const Location& destination) const

```

Comment:
Use override keyword: In the function declarations of isCorrectMove and getCharacter, use the override keyword to explicitly indicate that these functions are overriding base class functions. This can help improve code clarity and make it easier to understand the class hierarchy.